docs/en/log_management.rst

```diff
@@ -0,0 +1,49 @@
+.. include:: ../common/common_definitions.rst
+
+.. _Log_Management:
+
+Retention Policy
+================
+
+Log management of a OP and an RP
+------------------------------------
+
+OPs and RPs MUST retain the following. 
+
+1. A transaction log containing the exchanged messages. The messages stored in the log MUST be at least the following:
+
+    - **AuthenticationRequest**
+    - **AuthenticationResponse** related to the *AuthenticationRequest**.
+    - **TokenRequest** related to the *AuthenticationRequest*.
+    - **TokenResponse** related to the *TokenRequest*. 
+    - The **UserInfoRequest** related to the *TokenRequest*, if any.
+    - The **UserInfoResponse** related to the *UserInfoRequest*, if any.
+    - **RevocationRequest** related to the *TokenRequest*, if any.
+    - The **RevocationResponse** related to the *RevocationRequest*, if any.
+
+2. A federation log containing, for each *AuthenticationRequest*, the **Trust Chain** related to the entity which messages are being exchanged with. In the case of a log handled by an OP receiving an authentication request from an RP, it MUST contain:
+
+    - The **Entity Configuration** of the requesting RP.
+    - The **Entity Configuration** of the SA (in the case of an aggregated RP), if any.
+    - The **Entity Statement** of the SA referring to the RP, if any.
+    - The **Entity Configuration** of the TA.
+    - The **Entity Statement** of the TA referring to the RP.
+
+In the case of a registry handled by an RP making an authentication request to a OP, it MUST contain:
+
+    - The **Entity Configuration** of the OP.
+    - The **Entity Configuration** of the TA.
+    - The **Entity Statement** of the TA referring to the OP.
+
+.. warning::
+    The information stored in the logs MUST be retained and managed for not less than 24 months in full compliance with current privacy regulations. Access to the data MUST be restricted to designated personnel. In order to ensure confidentiality data encryption mechanisms or employed database systems (DBMS) that realize encrypted persistence of information MAY be adopted. Mechanisms that ensure integrity and non-repudiation MUST be put in place.
```

Comment:
National privacy regulations

we must say national, otherwise it would be not clear if those are internal to an org or to the federation policy or the italy, europe and so on ...

Reply:
encryption is mandatory following GDPR guidelines for log retention storage